_episodes/33.md

```diff
@@ -0,0 +1,266 @@
+  both request and response headers. We use a custom Lua filter to parse the 
+  request and extract the `x-trino-user` header. Then, we call the router 
+  service, which returns the upstream cluster address.
+
+## PR of the month: PR 8956 Add support for external db for schema management in mongodb connector
+
+This month's [PR of the month](https://github.com/trinodb/trino/pull/8956) comes
+from today's guest Siddhant to solve [this issue related to the MongoDB connector](https://github.com/trinodb/trino/issues/8887).
+
+Siddhant created the issue in response to the common issue that MongoDB 
+connector users face when they don't have write capability in the Mongo system.
+Since Mongo has no implicit schema, Trino uses a schema definition that is
+written to a special Mongo database. This PR enables users without write access
+to create an external location to store their schema to avoid this issue.
+
+Thanks Siddhant for raising this issue, as it's a common issue beginners using
+the MongoDB connector face commonly.
+
+## Bonus PR of the month: PR 8202 Metadata for alias in elasticsearch connector only uses the first mapping
+
+This bonus [PR of the month](https://github.com/trinodb/trino/pull/8202) comes
+from another one of today's guest Suman which ends up solving multiple issues,
+meaning this feature is in high demand! 
+
+The problem brought up by these issues also have to do with how we are mapping
+schemas over NoSQL databases that don't implicitely have a schema. In this case
+Elasticsearch stores it's schema in an object called a mapping. This mapping can
+be strict or dynamic for various portions of the document that gets inserted.
+The object that correlates to a table in Elasticsearch is called an index. To
+keep Elasticsearch fast, multiple indexes are created periodically to support a
+given document type similar to partitioning in a database. In general, these 
+index follow a very common mapping for a given type, but the reality is that 
+Elasticsearch allows you to vary from the mapping. Trino currently simplifies
+the way this is done by only reading the first mapping and assuming that all
+indexes and documents follow this schema. This pull request addresses this issue
+by scanning a much larger sample of mappings and merging the schema to handle
+any conflicts. It then goes further to cache these merged mappings for a given
+amount of time.
+
+Thanks for all of your continued work on this Suman! It will help a lot!
+
+## Demo of the month: Trino Fiddle: A tool for easy online testing and sharing of Trino SQL problems and their solutions
+
+This months demo will showcase a tool that Brian modified from [SQL Fiddle](http://sqlfiddle.com/)
+tool called Trino Fiddle. This tool will allow Trino users to share problems
+and answer questions that other Trino users are facing.
+
+This tool is currently being tested so it's not available for public use yet, 
```

Comment:
I would drop this. Or reword to make sure and omit promise about future release for now...